src/Processing.js

```diff
@@ -4066,10 +4066,39 @@
     p.noLoop = function() {
       doLoop = false;
       loopStarted = false;
-      clearInterval(looping);
+      
+      // requestAnimationFrame edit
+      //clearInterval(looping);
```

Comment:
don't leave old code in as a comment: the commit history takes care of making sure deleted code is easily restored.